docs/tutorials/alerting_based_on_metrics.md

```diff
@@ -3,16 +3,18 @@ title: Alerting based on metrics
 sort_rank: 5
 ---
 
+# Alerting based on metrics
```

Comment:
Ah I should have mentioned this - we don't need the H1 title in the Markdown itself anymore, see https://github.com/prometheus/docs/pull/2662.

Reply:
Addressed in [efb742e](https://github.com/prometheus/docs/pull/2667/commits/efb742e4961d0ad06695cef6dc8a9dbb27fcb05d)